lib/assembly-image/jp2_creator.rb

```diff
@@ -120,9 +121,10 @@ def make_tmp_tiff(tmp_folder: nil)
                        vips_image
                      end
 
-        tiff_image.tiffsave(tmp_path, bigtiff: true) # Use bigtiff so we can support images > 4GB
+        tiff_image.autorot.tiffsave(tmp_path, bigtiff: true) # Use bigtiff so we can support images > 4GB
```

Comment:
so this works the same as what Tony recommended?

```
Vips::Image.new_from_file(input_file_name, autorotate: true)
```

Are we sure this isn't rotating images that don't need it?

Reply:
> Are we sure this isn't rotating images that don't need it?

no, I am not sure, but this certainly sounds like what we want( https://www.libvips.org/API/current/libvips-conversion.html#vips-autorot) and it fixed the test I wrote and broke no others.

Reply:
https://www.rubydoc.info/gems/ruby-vips/Vips%2FImage.new_from_file it doesn't look like `autorotate` is a documented option to `.new_from_file`